src/sentry/static/sentry/app/components/links/link.tsx

```diff
@@ -15,7 +15,7 @@ type Props = {
   to: string | ToLocationFunction | LocationDescriptor;
   // Styles applied to the component's root
   className?: string;
-} & Omit<AnchorProps, 'href' | 'target'>;
+} & Omit<AnchorProps, 'href'>;
```

Comment:
I wouldn't remove the target from the Omit. The idea was that if need to open a link in a new tab you should use the component `<ExternalLink />`

Reply:
Talked with @billyvg about this, this is one of the few times we're opening a new page within our own application. External link was made for links to outside websites and takes a href and we need to use the RouterLink's to for this link to another place within our app.

I also asked the team if they meant for it to open in another tab, and it seems pretty much everyone wanted it to keep opening in a new window.

Reply:
Yeah, I think it's better to allow an exception in this case because 1) it opens a new tab to our app, 2) the URL would require stringify-ing the query string. But because it's an exception, would it be better to `@ts-ignore` where we use it and explain why?

Reply:
switched to ts-ignore for this case